src/openrct2/actions/MazePlaceTrackAction.cpp

```diff
@@ -119,7 +119,7 @@ GameActions::Result MazePlaceTrackAction::Query() const
 
     const auto& ted = GetTrackElementDescriptor(TrackElemType::Maze);
     money32 price = (((ride->GetRideTypeDescriptor().BuildCosts.TrackPrice * ted.Price) >> 16));
-    res.Cost = ((canBuild.Cost + price) / 2) * 10;
+    res.Cost = ToGBP(((canBuild.Cost + price) / 2));
```

Comment:
`price` and `canBuild.Cost` are both in terms of `money32` so this doesn't make sense to go through `ToGBP`. Either it should stay as *10 or they are not `money32`.